src/components-examples/angular/tabs/tab-group-lazy-loaded/tab-group-lazy-loaded-example.ts

```diff
@@ -0,0 +1,21 @@
+import { Component } from '@angular/core';
+
+/**
+ * @title Tab group where the tab content is loaded lazily (when activated)
+ * @order 50
+ */
+@Component({
+  selector: 'sbb-tab-group-lazy-loaded-example',
+  templateUrl: 'tab-group-lazy-loaded-example.html',
+})
+export class TabGroupLazyLoadedExample {
+  tabLoadTimes: Date[] = [];
```

Comment:
Caching loading times leads to a false assumption that content is rendered only once. In fact, every time a tab is activated, the content is reloaded (I tested this with a custom component, OnInit is called every time a tab gets activated).